Dockerfile

```diff
@@ -22,15 +22,21 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         automake \
         autoconf \
         m4 \
-        gperf && \
-    pip install ply && \
+        gperf \
+        wget && \
```

Comment:
seems like inconsistent placement of `&&`

Reply:
What do you mean?.. that line is installing `wget` along with other packages.